domain/src/Commands/DomainCommands.php

```diff
@@ -0,0 +1,1288 @@
+    }
+    else {
+      $this->logger()->warning(dt('!domain set to primary domain, but is also inactive.',
+        ['!domain' => $domain->getHostname()]));
+    }
+    return $domain->id();
+  }
+
+  /**
+   * Deactivates the domain.
+   *
+   * @param $domain_id
+   *   The numeric id or hostname of the domain to disable.
+   * @usage drush domain-disable example.com
+   * @usage drush domain-disable 1
+   *
+   * @command domain:disable
+   * @aliases domain-disable
+   *
+   * @return string
+   *   'disabled' if the domain is now disabled.
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  public function disable($domain_id) {
+    // Resolve the domain.
+    if ($domain = $this->getDomainFromArgument($domain_id)) {
+      if ($domain->status()) {
+        $domain->disable();
+        $this->logger()->info(dt('!domain has been disabled.',
+          ['!domain' => $domain->getHostname()]));
+      }
+      else {
+        $this->logger()->info(dt('!domain is already disabled.',
+          ['!domain' => $domain->getHostname()]));
+      }
+      if ($domain->status()) {
+        return 'enabled';
+      }
+      else {
+        return 'disabled';
+      }
+    }
+    return 'unknown domain';
+  }
+
+  /**
+   * Activates the domain.
+   *
+   * @param $domain_id
+   *   The numeric id or hostname of the domain to enable.
+   * @usage drush domain-disable example.com
+   * @usage drush domain-enable 1
+   *
+   * @command domain:enable
+   * @aliases domain-enable
+   *
+   * @return string
+   *   'enabled' if the domain is now enabled.
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  public function enable($domain_id) {
+    // Resolve the domain.
+    if ($domain = $this->getDomainFromArgument($domain_id)) {
+      if (!$domain->status()) {
+        $domain->enable();
+        $this->logger()->info(dt('!domain has been enabled.',
+          ['!domain' => $domain->getHostname()]));
+      }
+      else {
+        $this->logger()->info(dt('!domain is already enabled.',
+          ['!domain' => $domain->getHostname()]));
+      }
+      if ($domain->status()) {
+        return 'enabled';
+      }
+      else {
+        return 'disabled';
+      }
+    }
+    return 'unknown domain';
+  }
+
+  /**
+   * Changes a domain label.
+   *
+   * @param $domain_id
+   *   The numeric id or hostname of the domain to relabel.
+   * @param $name
+   *   The name to use for the domain.
+   * @usage drush domain-name example.com Foo
+   * @usage drush domain-name 1 Foo
+   *
+   * @command domain:name
+   * @aliases domain-name
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  public function renameDomain($domain_id, $name) {
+    // Resolve the domain.
+    if ($domain = $this->getDomainFromArgument($domain_id)) {
+      $domain->saveProperty('name', $name);
+      return $domain->name();
+    }
+    return 'unknown domain';
+  }
+
+  /**
```

Comment:
I think we should just delete this command.

Reply:
Sorry, got confused. Do you mean delete the "drush domain:disable" and "drush domain:enable" commands, or delete the "drush domain:machine-name" command which I commented out?

Reply:
machine name

Reply:
machine name

Reply:
machine name

Reply:
machine name